index.html

```diff
@@ -8,38 +8,116 @@
   <style>
     .header {
       text-align: center;
+     
     }
+
+    #font {
+      font-family: "Century Gothic";
+    }
+
+   .table thead th {
+      color:  #000099;
+   
+    }
+
+    
+
   </style>
 </head>
-<body>
+<body id = "font">
   <div class="container">
     <div class="header">
-      <h1>Week 1: HTML Tables</h1>
-      <p>We'll add our data to this table.</p>
+      <div class = "col-sm-4">
+      <a href = "http://mpdc.dc.gov/page/district-crime-data-glance" target="_blank">
```

Comment:
I like this choice. Note that for next week's lesson, we'll need something with more observations. You could still use the same data, but perhaps broken out by month or day?